pkg/internal/transform/route/cluster.go

```diff
@@ -45,7 +45,7 @@ func InitAutoClassifier() error {
 }
 
 //nolint:cyclop
-func ClusterPath(path string) string {
+func ClusterPath(path string, replacement string) string {
```

Comment:
Small nit here, can you please change the function second argument to be a char? That way we don't need to dereference the string multiple times.

Reply:
I ended up solving this a little differently with my last commit. If I were to use a rune parameter, it would necessitate another type casting to byte as within the `ClusterPath` function, a byte is required. So I thought it better to use a byte parameter instead and move the deferencing of the string to the `ClusterPath` call within `classifyFromPath`. I hope you're okay with that. Tests were adjusted to account for this change.